tests/settings.py

```diff
@@ -89,6 +89,15 @@
     "tests",
 )
 
+PASSWORD_HASHERS = [
+    "django.contrib.auth.hashers.PBKDF2PasswordHasher",
+    "django.contrib.auth.hashers.PBKDF2SHA1PasswordHasher",
+    "django.contrib.auth.hashers.Argon2PasswordHasher",
+    "django.contrib.auth.hashers.BCryptSHA256PasswordHasher",
+    "django.contrib.auth.hashers.ScryptPasswordHasher",
+    "tests.custom_hasher.MyPBKDF2PasswordHasher",
+]
+
```

Comment:
How about something like this rather than repeating the list of "standard" hashers:
```suggestion
+PASSWORD_HASHERS = django.conf.settings.PASSWORD_HASHERS + ["tests.custom_hasher.MyPBKDF2PasswordHasher"]
```